### PR TITLE
OBPIH-2742 Fix disappearing qty revised and reason code on edit page

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: src/js/components/stock-movement-wizard/outbound/EditPage.jsx [hunk 1]
```diff
@@ -266,6 +273,7 @@ class EditItemsPage extends Component {
   }
 
   setEditPageItems(response) {
+    console.log(this.state.values.editPageItems);
```
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/outbound/EditPage.jsx:276]: Console log

File: src/js/components/stock-movement-wizard/outbound/EditPage.jsx [hunk 2]
```diff
@@ -441,6 +449,18 @@ class EditItemsPage extends Component {
     return Promise.resolve();
   }
 
+  updateRow(values, index) {
+    const item = values.editPageItems[index];
```
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/outbound/EditPage.jsx:453]: Does this problem occur only on edit page items on outbound sm? How about add items pages after save or when you have stock list?

File: src/js/components/stock-movement-wizard/outbound/EditPage.jsx [hunk 3]
```diff
@@ -441,6 +449,18 @@ class EditItemsPage extends Component {
...
+    let val = values;
+    val = update(values, {
+      editPageItems: { [index]: { $set: item } },
+    });
+    console.log(index, item);
```
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/outbound/EditPage.jsx:458]: console.log

**Comment by jmiranda** [comment]
@DominikaHatala I requested some changes but then ended up doing them myself while I was adding the database migrations.

File: src/js/components/stock-movement-wizard/outbound/EditPage.jsx (hunk 2 above)
Comment (DominikaHatala) [reply src/js/components/stock-movement-wizard/outbound/EditPage.jsx:452]: @awalkowiak You're right. I've added this update method to add items page in all workflows.

**Pull request merged**